### PR TITLE
Remove unnecessary dependecies from dbnd airflow

### PR DESCRIPTION
**Pull request by FHoffmannCode**

### CODE REVIEW

File: plugins/dbnd-airflow-export/src/dbnd_airflow_export/logic.py [hunk 1]
```diff
@@ -325,8 +327,9 @@ def _get_task_instances_without_date(
 def _get_task_instances(
     since, dag_ids, quantity, dagbag, include_logs, include_xcom, session
 ):
+    query_fields = deepcopy(ETaskInstance.query_fields()).extend(EDagRun.query_fields())
```
Comment (jonathanshir) [inline plugins/dbnd-airflow-export/src/dbnd_airflow_export/logic.py:330]: Replace the asterix ("*") with `deepcopy`? Can you elaborate? :)

File: modules/dbnd-airflow/setup.py [hunk 2]
```diff
@@ -28,7 +28,7 @@
         airflow_1_10_9=requirements_for_airflow + ["apache-airflow==1.10.9"],
         airflow_1_10_10=requirements_for_airflow + ["apache-airflow==1.10.10"],
         airflow_1_10_11=requirements_for_airflow + ["apache-airflow==1.10.11"],
-        airflow_1_10_12=requirements_for_airflow + ["apache-airflow==1.10.12"],
+        airflow_1_10_12=["apache-airflow==1.10.12"],
```
Comment (jonathanshir) [inline modules/dbnd-airflow/setup.py:31]: What about other versions? Do they require the `requirements_for_airflow` necessarily?

**Review by jonathanshir — CHANGES_REQUESTED** [review]
Tested on py2 only, or on py3 as well?

File: plugins/dbnd-airflow-export/src/dbnd_airflow_export/logic.py (hunk 1 above)
Comment (FHoffmannCode) [reply plugins/dbnd-airflow-export/src/dbnd_airflow_export/logic.py:330]: in py27 you cannot do this

```python
foo(*iterable, *iterable2)
```

therefore we need to construct one iterable with all query fields.

I use `deepcopy` for `ETaskInstance.quey_fields()` so I dont modify the original db model  instance with 

```python
ETaskInstnce.query_fields().extend(EDagRun.query_fields())
```

which could result in some strange and difficult to track bugs

File: plugins/dbnd-tensorflow/src/dbnd_tensorflow/_plugin.py [hunk 3]
```diff
@@ -19,12 +19,12 @@ def dbnd_setup_plugin():
     )
 
     register_marshaller(
-        tensorflow.python.keras.engine.training.Model,
+        tensorflow.keras.models.Model,
```
[the remark this answers is not in the document]
Comment (FHoffmannCode) [reply plugins/dbnd-tensorflow/src/dbnd_tensorflow/_plugin.py:22]: ok

File: plugins/dbnd-airflow-export/src/dbnd_airflow_export/logic.py (hunk 1 above)
Comment (jonathanshir) [reply plugins/dbnd-airflow-export/src/dbnd_airflow_export/logic.py:330]: Resolved, great find!

**Comment by FHoffmannCode** [comment]
@jonathanshir tested on py27 and py3 of course

File: modules/dbnd-airflow/setup.py (hunk 2 above)
Comment (FHoffmannCode) [reply modules/dbnd-airflow/setup.py:31]: I think its better if we let them be. All versions lower than 1.10.12 use cattrs 0.9 and even though I tested them with py27 and they all seem to work fine I think its safer to leave them as they are and use 1.10.12.

To be honest in future we should gradually drop support for older airflow versions and port all clients to 1.10.12 and higher

Comment (jonathanshir) [reply modules/dbnd-airflow/setup.py:31]: This is definitely the plan, users will be migrating to 1.10.12

**Pull request closed without merging**